tests/scripts/t3000/run_t3000_frequent_tests.sh

```diff
@@ -92,6 +92,7 @@ run_t3000_trace_stress_tests() {
   echo "LOG_METAL: Running run_t3000_trace_stress_tests"
 
   NUM_TRACE_LOOPS=30 pytest tests/ttnn/unit_tests/test_multi_device_trace.py
+  NUM_TRACE_LOOPS=30 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest tests/ttnn/unit_tests/test_multi_device_trace.py
```

Comment:
Can we run this test as an extra parameter for pytest instead of invoking test_multi_device-trace.py twice here?

Reply:
The issue here is that the `WH_ARCH_YAML` parameter takes effect during the `t3k_device_mesh` test fixture initialization. The other option would be to add a new test fixture that inherently uses this env-var, but that would also require a repeated reference to this test script.

Reply:
Ok, I think this would be a far too complicated change then out of scope for this PR.
Can we make a follow-up issue here so we can remove setting ENV variables for these pytests and pass as input parameters instead?

Reply:
Being tracked here: https://github.com/tenstorrent/tt-metal/issues/9514